American Whitewater/Extensions/UIColor+Hex.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  UIColor+Hex.swift
+//  Duffek Mobile
+//
+//  Created by David Nelson on 8/22/18.
+//  Copyright © 2019 Duffek Mobile. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIColor {
+    
+    // List of BrewFund Specific Colors
```

Comment:
What is BrewFund?

Reply:
My other company. I copied some code over from our other app and didn't refactor :) This is fixed.